src/interfaces.ts

```diff
@@ -69,6 +69,10 @@ export type Column = Array<ColumnDescriptor>;
 
 export type Row = Array<string | number | boolean>;
 
+export interface WriteQueryResult {
+  data: null;
```

Comment:
If this is always going to be `null`, then we just shouldn't return anything in any externally facing APIs to user-land. That just ends up being confusing. Not sure if this _is_ the case, I'll see as I read on. But if we're _ever_ returning this to user land, we should just return `void` in typescript speak, which is effectively `undefined`.